client/src/components/files/FileCard.vue

```diff
@@ -36,28 +47,35 @@
         <span>{{ timeSince(file.updated, '--', 'short') }}</span>
       </ion-text>
     </div>
-  </div>
+  </ion-item>
 </template>
 
 <script setup lang="ts">
 import { Formatters, FormattersKey } from '@/common/injectionKeys';
 import { EntryStat } from '@/parsec';
-import { IonAvatar, IonIcon, IonText, IonTitle } from '@ionic/vue';
+import { IonAvatar, IonCheckbox, IonIcon, IonText, IonTitle } from '@ionic/vue';
```

Comment:
Missing `IonItem` import. Always keep you dev tools open in your browser :p

Reply:
Sure, done it ^^